README.md

```diff
@@ -86,7 +86,7 @@ supported by `ring`. At the time of writing, this means x86, x86-64, aarch64,
 armv7, powerpc64le, riscv64gc and s390x. For more information, see [the
 supported `ring` CI targets][ring-ci-targets].
 
-Rustls requires Rust 1.60 or later.
+Rustls requires Rust 1.61 or later.
```

Comment:
edit this also in src/lib.rs

Reply:
(It was actually there, just in the other commit -- fixed that up now.)